code/modules/cooking/machinery/foodcart.dm

```diff
@@ -0,0 +1,147 @@
+/obj/machinery/food_cart
+	name = "food cart"
+	desc = "A compact unpackable mobile cooking stand. Wow! When unpacked, it reminds you of those greasy gamer setups some people on NTNet have."
+	icon = 'icons/obj/machinery/cooking_machines.dmi'
+	icon_state = "foodcart"
+	density = TRUE
+	anchored = FALSE
+	stat = POWEROFF
+	use_power = POWER_USE_OFF
+	req_access = list(ACCESS_KITCHEN)
+	var/unpacked = FALSE
+	// Stuff that's gonna make up the food stand when unpacked. Self-explanatory.
+	var/obj/machinery/appliance/cooker/grill/stand/cart_griddle
+	var/obj/machinery/smartfridge/foodheater/stand/cart_smartfridge
+	var/obj/structure/table/reinforced/cart_table
+	var/obj/effect/food_cart_stand/cart_tent
+	var/list/packed_things
```

Comment:
😩

Reply:
this looks and feels like it will be buggy af (but I didn't test it in game)

Reply:
Oh, believe me. I've tested this thoroughly. And painfully.